vllm/model_executor/models/molmo.py

```diff
@@ -461,30 +461,71 @@ def forward(
         return output
 
 
-class MolmoMLP(nn.Module):
+class SwiGLU(nn.Module):
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        x, gate = x.chunk(2, dim=-1)
+        # Note that the order is reversed compared to
+        # SiluAndMul.
+        return x * F.silu(gate)
```

Comment:
`SiluAndMul` will use custom_ops through `forward_cuda` on GPU, so perhaps we should still use it for better performance, especially this is used in llm backbone?

I think it worths to compare the performance between `reshape->SiluAndMul` and native SwiGLU to find the better one.

Reply:
Currently there should be 3 models in vllm that use this activation function: [jais](https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/models/jais.py#L51), [ultravox](https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/models/ultravox.py#L203) and molmo. I'm thinking of implementing a custom kernel in the following  PR, and keep things unchanged for now.